lib/cabal-project-parser.nix

```
@@ -68,7 +68,7 @@ let
               drv)
       ) + subdir)
       (if repo ? subdir
-        then builtins.map (x: "/" + x) (
+        then builtins.map (x: if x == "." then "" else "/" + x) (
```

```suggestion
        then builtins.map (x: if x == "." then "./" else "/" + x) (
```
might be easier to understand?